pkgs/top-level/python-packages.nix

```diff
@@ -1221,7 +1221,7 @@ in {
   };
 
   batinfo = buildPythonPackage rec {
-    version = "0.3";
+    version = "-1.3";
```

Comment:
This is not right; the version must start with a digit.

Reply:
typically typo mistake too. I will mark the PR as RFC once I make sure everything works. My last commit "Now works" was a shout of joy after sphalerite helped me solve sthg I 'd spent quite some time. thanks for the review